assets/js/selecting.js

```diff
@@ -0,0 +1,61 @@
+function isSSLothable(v) {
+	var m = v.major;
+	var n = v.nver;
+	return (
+		(m == 4 && n == 37) ||
+		(m == 5 && n == 38) ||
+		(m == 6 && n == 39) ||
+		(m == 7 && n == 40) ||
+		(m == 8 && n == 41) ||
+		(m == 9 && n == 42) ||
+		(m == 10 && n == 43) ||
+		(m == 11 && n == 43) ||
+		(m == 12 && n == 44) ||
+		(m == 13 && n == 45)
+	);
+}
+
+function getResultText(v) {
+	if (v.major == 0) {
+		return "Ver. 0.0.0-0 indicates a developer unit. This guide only works with retail units.";
```

Comment:
I would just change this message to "It doesn't look like you input a valid version" or something like that because since Sys 0 0 0 0 U is the default option people might click the button and just get confused by this message.

People who actually have a developer unit can hopefully redirect themselves to NH

Reply:
I wrote the message like that on purpose, so they see the version they don't have and then select the correct version, but if you want it like that ill change it

Reply:
Second thought, good idea